### ISSUE TITLE
MekHQ 0.3.31 Personnel Market update blank

### ISSUE THREAD

**Issue by Nitedawn** [MegaMek/mekhq#131]
Hi Guys,

Daily log shows the personnel market has been updated, but the market is blanket.  Add (GM) button wont add personnel either.

Save and mekhqlog.txt upload.
[Archive.zip](https://github.com/MegaMek/mekhq/files/402851/Archive.zip)

Cheers

**Comment by neoancient** [comment]
The personnel market you are using (AtB) has a 1/6 chance of not generating any hire in a given week (as does the StratOps method on which it is based). The log shows that the market is updated whenever it is checked, even if the update does not change the contents. I advanced to the next week of your campaign and it generated an infantry soldier with platoon, so I don't see a problem there. The "Add(GM)" button does not add personnel to the market, but does a GM add of the selected hire; that is, adds the person to the campaign without paying any normal hiring costs.

I did notice and fix a problem when the Add(GM) button is clicked with no hire selected, but the only effect it had was reporting an error in the log file.
